spec/requests/green_lanes/moving_requirements_controller_spec.rb

```diff
@@ -52,14 +52,20 @@
         }
       end
 
-      it { is_expected.to redirect_to(result_green_lanes_check_moving_requirements_path) }
+      it do
+        expect(make_request).to redirect_to(result_green_lanes_check_moving_requirements_path(
```

Comment:
Shouldn't this be testing the `response` object ?

Reply:
I fixed the existing test, I think we are more interested in the redirection here.